src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -166,6 +166,10 @@ class SourcesTree extends Component {
   }
 
   getIcon(item, depth) {
+    if (item.path === "/webpack://") {
+      return <Svg name="webpack" className="webpack-icon" />;
```

Comment:
do we need this: `className="webpack-icon"`?

Reply:
No we don't, nice catch